kubernetes/client.go

```diff
@@ -99,30 +99,24 @@ func NewClient() (*IstioClient, error) {
 
 // It returns a list of all namespaces/projects of the cluster.
```

Comment:
if there are no projects in the code, docs should not mention them either :)

Reply:
Changed! I think it was the only occurrence in our source code ;D